pontoon/homepage/views.py

```diff
@@ -28,6 +30,23 @@ def homepage(request):
     else:
         start_url = reverse('pontoon.teams')
 
-    return render(request, 'homepage.html', {
+    playground_url = reverse('pontoon.translate.locale.agnostic', kwargs={
+        'slug': 'tutorial',
+        'part': 'playground',
+    })
+
+    intro_url = reverse('pontoon.translate.locale.agnostic', kwargs={
+        'slug': 'pontoon-intro',
+        'part': 'messages.properties',
+    })
+
+    homepage = Homepage.objects.all()[0]
```

Comment:
That's the same as `Homepage.objects.first()`. But you really want `Homepage.objects.last()` if you change ordering.